draft-ietf-scitt-architecture.md

```diff
@@ -167,27 +167,15 @@ SCITT achieves this by having producers, auditors, etc. (also called Issuers) pu
 
 {::boilerplate bcp14-tagged}
 
-# Use Cases
-
-The building blocks defined in SCITT are intended to support applications in any supply chain that produces or relies upon digital artifacts, from the build and supply of software and IoT devices to advanced manufacturing and food supply.
-
-## Relation to Certificate Transparency
-
-SCITT is a generalization of Certificate Transparency {{-CT}}, which can be interpreted as a transparency architecture for the supply chain of X.509 certificates.
-Considering CT in terms of SCITT:
-
-- CAs (Issuers) sign X.509 TBSCertificates (Artifacts) to produce X.509 certificates (Signed Statements)
-- CAs submit the certificates to one or more CT logs (Transparency Services)
-- CT logs produce Signed Certificate Timestamps (Transparent Statements)
-- Signed Certificate Timestamps are checked by Verifiers.
-- The Append-only Log can be checked by Verifiers and Auditors
-
 # Terminology {#terminology}
 
 The terms defined in this section have special meaning in the context of Supply Chain Integrity, Transparency, and Trust, which are used throughout this document.
 When used in text, the corresponding terms are capitalized.
 To ensure readability, only a core set of terms is included in this section.
 
+The label "394" is expected to be reserved by this document, in the COSE Header Parameters Registry.
+Please remove this comment before publishing.
```

Comment:
Why is this in the terminology?

Reply:
its a description of the CBOR labels we use in examples, which are "requested" and are not actually assigned.